core/geotime/src/main/java/mil/nga/giat/geowave/core/geotime/store/filter/SpatialQueryFilter.java

```diff
@@ -196,8 +204,73 @@ public void fromBinary(
 					fieldId));
 		}
 		buf.get(theRest);
-		queryGeometry = geometryInterner.intern(GeometryUtils.geometryFromBinary(geometryBinary));
+		// cannot intern the prepared geometry. It uses a Object.hashCode().
+		preparedGeometryImage = geometryImageInterner.intern(new GeometryImage(
+				geometryBinary));
+		preparedGeometryImage.init();
```

Comment:
Is there a good reason to have a separate .init() method?   Since there's no parameterless constructor for GeometryImage  you would only ever be calling init() directly after a constructor call (I don't see any benefit to deferring the init;  in that case you would just deferr object construction all together).

So thoughts on making the init() method private and calling it explicitly in the GeometryImage constructor(s)?